google-cloud-spanner/lib/google/spanner/v1/spanner_pb.rb

```diff
@@ -10,7 +10,6 @@
 require 'google/protobuf/timestamp_pb'
 require 'google/spanner/v1/keys_pb'
 require 'google/spanner/v1/mutation_pb'
-require 'google/spanner/v1/query_plan_pb'
```

Comment:
this file is also required in `result_set_pb`, which is the next file required. This is probably okay.